### DISCUSSION TITLE
Laser damage diminishes with distance

### DISCUSSION THREAD

**Discussion by Indy2222** [DigitalExtinction/Game#164] [Ideas]
Consider coming up with a formula which diminishes laser damage with distance from the muzzle.

**Comment by JackCrumpLeys** [comment]
something like: damage = base_damage - (distance_from_muzzle * damage_loss_factor)

In this equation, "base_damage" is the maximum damage that the laser weapon can deal at point-blank range, "distance_from_muzzle" is the distance between the muzzle of the laser weapon and the target, and "damage_loss_factor" is a constant value that determines how quickly the damage decreases with distance.

For example, if the base damage of the laser weapon is 100, and the damage loss factor is 0.5, the damage at a distance of 10 meters from the muzzle would be:

damage = 100 - (10 * 0.5) = 95

At a distance of 20 meters, the damage would be:

damage = 100 - (20 * 0.5) = 90

Should we calculate the loss from the range?
